### PR TITLE
Avoid JS assertEqual warnings, & use npx in tutorial text

### PR DESCRIPTION
**Pull request by matthewadams**

### CODE REVIEW

File: content/docs/guides/10-minute-tutorial.md [hunk 1]
```diff
@@ -449,7 +449,7 @@ gradle cucumber
 npm test
 
 # Run standalone
-./node_modules/.bin/cucumber-js
+npx cucumber-js
```
Comment (olleolleolle) [inline content/docs/guides/10-minute-tutorial.md:452]: https://www.npmjs.com/package/npx is a convenience package

> Executes `<command>` either from a local `node_modules/.bin`, or from a central cache, installing any packages needed in order for `<command>` to run.

Comment (olleolleolle) [reply content/docs/guides/10-minute-tutorial.md:452]: In npm 7.x, it's built-in – I looked that up, and found this page in the docs:

- https://docs.npmjs.com/cli/v7/commands lists it https://docs.npmjs.com/cli/v7/commands/npx
- https://docs.npmjs.com/cli/v6/commands does not list it, and I guess one has to use the npm-installable package at https://www.npmjs.com/package/npx

**Review by olleolleolle — COMMENTED** [review]
Super, thanks for improving the documentation!

I made notes on the `npx` tool, and I ask myself "how well-known should I expect npx to be, for the general Cucumber user?" Perhaps I'm off-base when my first thought was "do people know how to install that?" and "how do I teach the user whether they need to install it, or how they can tell that it's already built-in to npm?"

Perhaps this needs PR _no change_ and is 💯 .

**Comment by aslakhellesoy** [comment]
Hi @matthewadams,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**